ctsit/redcap_oncore_client
# Feature/sa scheduled send

Introduces sending of SA data as a cron job. Sends at an admin-defined interval.

Also updates coded variables for gender and mapping for OnCore Study date.

To test:
Create a new project from the updated template  
**OR**  
erase all data from your existing project and change the coded variables for gender as follows: 

|Old|New|
|:--|--:|
|F|1|
|M|2|
|B|9|

- Upload the new `summary_accrual_test_data`
- Go to the new "Configure Summary Accrual" plugin page at the project level
- Go to the "Upload Summary Accrual" plugin page and continually refresh. A new entry should appear within 5 minutes
